controller.py

```diff
@@ -73,6 +76,66 @@ def authenticated_client_from_request(_db, required=True):
     return INVALID_CREDENTIALS
 
 
+class IndexController(object):
+
+    def __init__(self, _db):
+        self._db = _db
+
+    def opds_catalog(self):
+        url = ConfigurationSetting.sitewide(self._db, Configuration.BASE_URL_KEY).value
+        catalog = dict(
+            id=url,
+            title='Library Simplified Metadata Wrangler',
+        )
+
+        catalog['links'] = [
+            {
+                "rel": "register",
+                "href": "/register",
+                "type": "application/opds+json;profile=https://librarysimplified.org/rel/profile/metadata-service",
+                "title": "Register your OPDS server with this metadata service"
+            },
+            {
+                "rel": "http://librarysimplified.org/rel/metadata/lookup",
+                "href": "/lookup{?urn*}",
+                "type": "application/atom+xml;profile=opds-catalog",
+                "title": "Look up metadata about one or more specific items",
+                "templated": "true"
+            },
+            {
+                "rel": "http://opds-spec.org/sort/new",
+                "href": "/{collection_metadata_identifier}/updates",
+                "type": "application/atom+xml;profile=opds-catalog",
+                "title": "Recent changes to your tracked collection",
```

Comment:
I'd change all these `title`s to "one of your tracked collections" to make it clear that you can track multiple collections.